packs/chef/actions/chef_client.py

```diff
@@ -8,14 +8,14 @@ class ChefClientRunner(ChefRunner):
     ChefRunner type implementation.
     Invokes chef-client binary with given arguments.
     '''
-
-    chef_binary = 'chef-client'
```

Comment:
Not, sure why) There are different values for this variable now. It couldn't be a reason though, just decided that it's more neat.